tests/integ/modin/conftest.py

```diff
@@ -79,8 +79,12 @@ def auto_annotate_sql_counter(request):
 
 @pytest.fixture(autouse=True)
 def check_sql_counter_invoked(request):
+    from tests.conftest import SKIP_SQL_COUNT_CHECK
+
```

Comment:
since we have moved sql_counter for snowpark oython usage, you probably need to bump this up to snowpark python level to allow skip sql counter check at snowpark python level

Reply:
that's already done here, right. We are importing from `tests.conftest` instead of `tests.modin.conftest`

Reply:
hmm... I understand your point here but shouldn't the tests fail if that is the case?

Reply:
I see what you mean here. If we add sql counter check to snowpark python and also run those tests in do multithreaded run with them, then, this needs to move up.